integrated_channels/moodle/exporters/learner_data.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+Learner data exporter for Enterprise Integrated Channel Moodle.
+"""
+
+from logging import getLogger
+
+from django.apps import apps
+
+from enterprise.api_client.discovery import get_course_catalog_api_service_client
+from integrated_channels.integrated_channel.exporters.learner_data import LearnerExporter
+from integrated_channels.utils import parse_datetime_to_epoch_millis
+
+LOGGER = getLogger(__name__)
+
+
+class MoodleLearnerExporter(LearnerExporter):
+    """
+    Class to provide a Moodle learner data transmission audit prepared for serialization.
+    """
+
+    def get_learner_data_records(
+            self,
+            enterprise_enrollment,
+            completed_date=None,
+            is_passing=False,
+            **kwargs
+    ):  # pylint: disable=arguments-differ,unused-argument
+        """
+        Return a MoodleLearnerDataTransmissionAudit with the given enrollment and course completion data.
+        If completed_date is None, then course completion has not been met.
+        If no remote ID can be found, return None.
+        """
+        completed_timestamp = None
+        if completed_date is not None:
+            completed_timestamp = parse_datetime_to_epoch_millis(completed_date)
```

Comment:
I didn't even know about this util. Nice.

Reply:
(tbh I learned about it while implementing Canvas and seeing some other channel do it)